### PR TITLE
update text of progress bar timing

### PR DESCRIPTION
**Pull request by segus3088**
## Description

Completing progress line in Spanish

## Specific Changes proposed

Completing progress line in Spanish
## Requirements Checklist
- [ ] Feature implemented / Bug fixed
- [ ] If necessary, more likely in a feature request than a bug fix
  - [ ] Change has been verified in an actual browser (Chome, Firefox, IE)
  - [ ] Unit Tests updated or fixed
  - [ ] Docs/guides updated
  - [ ] Example created ([starter template on JSBin](https://codepen.io/gkatsev/pen/GwZegv?editors=1000#0))
- [ ] Reviewed by Two Core Contributors

### CODE REVIEW

File: lang/es.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
   "Seek to live, currently behind live": "Buscar en vivo, actualmente demorado con respecto a la transmisión en vivo",
   "Seek to live, currently playing live": "Buscar en vivo, actualmente reproduciendo en vivo",
   "Progress Bar": "Barra de progreso",
-  "progress bar timing: currentTime={1} duration={2}": "{1} de {2}",
+  "progress bar timing: currentTime={1} duration={2}": "Barra de progeso: Transcurrido{1} de {2}",
```
Comment (mister-ben) [inline lang/es.json:33]: This was correct as "{1} de {2}". This one is a bit of a special case, as it's a translation with variable substitution. In English, it becomes "{1} of {2}".

**Review by mister-ben — CHANGES_REQUESTED** [review]

File: lang/es.json (hunk 1 above)
Comment (segus3088) [reply lang/es.json:33]: It is to make the translation look better

Comment (gkatsev) [reply lang/es.json:33]: In addition, a screen reader trying to try to read out that entire phrase will fall behind by quite a lot compared to it already likely falling being just reading `{1} of {2}`.
The translations should match what's here https://github.com/videojs/video.js/blob/main/lang/en.json#L17

Comment (gkatsev) [reply lang/es.json:33]: @segus3088 if you can revert this line change we'd be happy to merge it in. Thanks!

Comment (segus3088) [reply lang/es.json:33]: @gkatsev look better "Barra de progreso: {1} de {2}"

Comment (gkatsev) [inline lang/es.json:33]:
```suggestion
  "progress bar timing: currentTime={1} duration={2}": "{1} de {2}",
```

**Review by gkatsev — APPROVED** [review]

**Review by bcbclifford — APPROVED** [review]

**Pull request merged**